wall_of_wisdom/map.conf

```
@@ -0,0 +1,13 @@
+team.1.pos = (-5,8,94)
+r = 115
+initial_stuff = Default:cobble 50,default:torch 20,default:pick_steel
```

```suggestion
initial_stuff = default:cobble 50,default:torch 20,default:pick_steel
```
You shouldn't use capital letters else that will not give the correct item.